ouroboros-consensus-cardano/tools/db-analyser/HasAnalysis.hs

```diff
@@ -34,6 +37,8 @@ class (HasAnnTip blk, GetPrevHash blk) => HasAnalysis blk where
   -- | Emit trace markers at points in processing.
   emitTraces     :: WithLedgerState blk -> [String]
 
+  blockStats     :: blk -> [String]
```

Comment:
Perhaps a comment explaining that this was introduced for the sake of the `BenchmarkLedgerOps` analysis pass.

Reply:
Absolutely! I overlooked this :/ Thanks.